### PR TITLE
[Misc] Update credits

### PR DESCRIPTION
**Pull request by DayKev**
Edit: Credits doc should be mostly complete now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,93 +37,6 @@ For detailed guidelines on documenting your code, refer to the [comments.md](./d
 Check out [Github Issues](https://github.com/pagefaultgames/pokerogue/issues) to see how can you help us!
 
 # 📝 Credits
-> If this project contains assets you have produced and you do not see your name here, **please** reach out.
+> If this project contains assets you have produced and you do not see your name here, **please** reach out, either [here on GitHub](https://github.com/pagefaultgames/pokerogue/issues/new) or via [Discord](https://discord.gg/pokerogue).
```
Comment (MokaStitcher) [inline README.md:40]:
```suggestion
> If this project contains assets you have produced and you do not see your name, **please** reach out, either [here on GitHub](https://github.com/pagefaultgames/pokerogue/issues/new) or via [Discord](https://discord.gg/pokerogue).
```

**Comment by Blitz425** [comment]
Can you add Leparagon under Trainer Sprites? He was the one who provided me with a custom Rose sprite for the Evil Team addition. I requested him to be added awhile back and I guess the wiki people forgot :S 
image

File: README.md (hunk 1 above)
Comment (DayKev) [reply README.md]: Whoops, made that change and then realized I should move everything to its own file since it's (hopefully) about to get much larger.

**Comment by flx-sta** [comment]
- flx-sta
is fine

**Comment by CodeTappert** [comment]
CodeTappert
- German Translation
- Localization Dev Work
- Evil Team Admins (up to Gen 6)

**Comment by Blitz425** [comment]
Typing on behalf of the other balance members for credits:

damocleas
Blitzy (Kazapple)
Cynthia_calliope
Esca
Fontbane
Plasto
Sethcurry
Starkrieg

Past Balancers
Swizzo
Zaccie

**Comment by Snailman11** [comment]
Snailman
- Bug Report Management relevant section

File: CREDITS.md [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+- Chapybara
+- PeachFresca
+
+### 🇰🇷 Korean (ko)
+- Enoch
+- KimJeongSun
+- Returntoice
+- sodamelon
+
+### 🇵🇹 Portuguese (pt-BR)
```
Comment (Adri1) [inline CREDITS.md:252]:
```suggestion
### 🇧🇷 Portuguese (pt-BR)
```

**Review by Adri1 — COMMENTED** [review]
Changed the Portugese flag by the Brazilian flag for pt-BR

File: CREDITS.md [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+
+### 🇰🇷 Korean (ko)
+- LeKaaN
+- Returntoice
+- sodamelon
+
+### 🇵🇱 Polish (pl)
+- Talo
+
+### 🇵🇹 Portuguese (pt-BR)
```
Comment (Adri1) [inline CREDITS.md:296]:
```suggestion
### 🇧🇷 Portuguese (pt-BR)
```

File: README.md [hunk 4]
```diff
@@ -37,93 +37,6 @@ For detailed guidelines on documenting your code, refer to the [comments.md](./d
...
-  - bizcoeindoloro
-  - mangalos810
-  - Involuntary-Twitch
-  - selstar
-  - koda_want_to_sleep
-  - thedreadedden
-
-### 🎨 Move Animations
-  - Pokémon Reborn
+Thank you to all the wonderful people that have contributed to the PokeRogue project! You can find the credits [here](./CREDITS.md).
```
Comment (Madmadness65) [inline README.md:42]:
```suggestion
Thank you to all the wonderful people that have contributed to the PokéRogue project! You can find the credits [here](./CREDITS.md).
```

**Review by Madmadness65 — COMMENTED** [review]
Just a minor nit, otherwise looks good to go.

File: CREDITS.md [hunk 5]
```diff
@@ -0,0 +1,407 @@
...
+## Reddit Moderators
+- TheZigglez
+- Vicksin
+- Sapphire
+- Javi
+- roi
+
+## External Tools
+- Ydarissep (RogueDex)
+- Admiral-Billy (Offline App - Desktop)
```
Comment (Tempo-anon) [inline CREDITS.md:407]: Any news from Red and their app?

File: CREDITS.md [hunk 6]
```diff
@@ -0,0 +1,407 @@
...
+- ImperialSympathizer
+- wormhood
+- gerolau
+- otterwatch
+
+## Trainer Portraits
+- pkmn_realidea (Paid Commissions)
+
+## Pokemon Sprites and Animation
+In addition to the list below, please check [the PokeRogue wiki](https://wiki.pokerogue.net/credits:credits) for a full list of Pokemon Sprite credits.
```
Comment (Tempo-anon) [inline CREDITS.md:76]: Accented e here for consistency?

File: CREDITS.md [hunk 7]
```diff
@@ -0,0 +1,407 @@
...
+- snoozbuster
+- zaccie
+- happinyz
+- PyGaVS
+- mcmontag
+- ElizaAlex
+- AsdarDevelops
+- Vassiat
+- RedstonewolfX
+- Sam/Flashfyre (initial developer, started PokeRogue)
```
Comment (Tempo-anon) [inline CREDITS.md:214]: Accent on the e

File: CREDITS.md [hunk 8]
```diff
@@ -0,0 +1,407 @@
...
+- thedreadedden
+- ImperialSympathizer
+- wormhood
+- gerolau
+- otterwatch
+
+## Trainer Portraits
+- pkmn_realidea (Paid Commissions)
+
+## Pokemon Sprites and Animation
```
Comment (Tempo-anon) [inline CREDITS.md:75]: Pokemon with accent e?

**Review by Tempo-anon — COMMENTED** [review]
Some more missing é

File: CREDITS.md (hunk 5 above)
Comment (MokaStitcher) [reply CREDITS.md:407]: Good question. I think they were given the junior dev status for it, but maybe External Tools would be best way for it to go indeed 🤔

Comment (DayKev) [reply CREDITS.md:407]: Added. Also send him a DM asking if he'd like to be credited under a different username just in case.

File: CREDITS.md [hunk 9]
```diff
@@ -99,11 +98,225 @@ In addition to the list below, please check [the PokéRogue wiki](https://wiki.p
 - Fleimer_
 - bizcoeindoloro
 - mangalos810
-- Involuntary-Twitch
 - selstar
-- koda_want_to_sleep
 - thedreadedden
```
Comment (MokaStitcher) [inline CREDITS.md:102]: I think this is Kieran, maybe it should be removed from here and that handle used to replace the rival_kieran one below?

**Review by Tempo-anon — DISMISSED** [review]

File: CREDITS.md [hunk 10]
```diff
@@ -99,11 +98,225 @@ In addition to the list below, please check [the PokéRogue wiki](https://wiki.p
...
+- Squip
+- TeraVolt
+- TheAetherPlayer
+- TheCynicalPoet
+- Tooni
+- TrainerSplash
+- Travis
+- Turtleye
+- Tyrell D. Barnes
+- \_vari\_
```
Comment (MokaStitcher) [inline CREDITS.md:186]: Other places below credit Vari as `“Vari” \_vari\_`
So maybe should be changed to that for consistency

**Review by MokaStitcher — APPROVED** [review]

**Review by Tempo-anon — APPROVED** [review]

**Pull request merged**